posthog/api/test/test_feature_flag.py

```diff
@@ -1332,3 +1332,23 @@ def assert_feature_flag_activity(self, flag_id: Optional[int], expected: List[Di
         self.assertEqual(
             activity, expected,
         )
+
+    def test_patch_api_as_form_data(self):
+        another_feature_flag = FeatureFlag.objects.create(
+            team=self.team,
+            name="some feature",
+            key="some-feature",
+            created_by=self.user,
+            filters={"groups": [{"properties": [], "rollout_percentage": 100}], "multivariate": None},
+        )
+        response = self.client.patch(
+            f"/api/projects/{self.team.id}/feature_flags/{another_feature_flag.pk}/",
+            data={"active": False},
+            content_type="application/x-www-form-urlencoded",
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(FeatureFlag.objects.get(pk=another_feature_flag.pk).name, "some feature")
```

Comment:
I guess we only really need the assertion that the response is 200

should we assert that active is now false?

Reply:
Checking for change to active showed it wasn't being changed. Django was throwing away the json passed in. Shifted the test to sending url form encoded format data and the changes were applied by django